plugins/inputs/google_cloud_storage/README.md

```diff
@@ -70,7 +70,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 The example output
 
-```shell
+```text
 2022-09-17T17:52:19Z I! Starting Telegraf 1.25.0-a93ec9a0
```

Comment:
This section actually doesn't show any sample metrics.. Maybe remove it?

Reply:
Fixed. https://github.com/influxdata/telegraf/pull/12989/commits/479ce69e2c42f6e58389202b91f63c3f79bf5413